gradle/libs.versions.toml

```
@@ -39,6 +39,7 @@ fragment-ktx = "1.8.9"
 glance = "1.1.1"
 googleServices = "4.4.4"
 guava = "33.4.6-android"
+healthConnectTesting = "1.0.0-alpha01"
```

```suggestion
healthConnectTesting = "1.0.0-alpha03"
```
Why not using the latest alpha? API version ? if so please add a comment next to the version.